### DISCUSSION TITLE
Running opensnitchd with `proc` give anyway ebpf error..

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#1233] [Q&A] [answered]
Hi, 

Following this problem https://github.com/evilsocket/opensnitch/issues/774#issuecomment-2520864679

I've set the `ProcMonitorMethod` to `proc`

but I still get 
<details><summary>EBPF error in the log</summary>
WAR -- nftables: error applying changes: conn.Receive: netlink receive: no such file or directory<br>
WAR -- error deleting system table: mangle-inet<br>
WAR -- nftables: error applying changes: conn.Receive: netlink receive: no such file or directory<br>
WAR -- error deleting system table: filter-inet<br>
IMP -- Start writing logs to /var/log/opensnitchd.log<br>
IMP -- [nftables] AddSystemRules() fw disabled<br>
WAR -- EBPF-DNS: Failed to attach uprobe uretprobe/gethostbyname : cannot open uprobe_events: open /sys/kernel/debug/tracing/uprobe_events: no such file or directory, (/lib/aarch64-linux-gnu/libc.so.6, 1016320)<br>
WAR -- EBPF-DNS: Failed to attach uprobe uprobe/getaddrinfo : cannot open uprobe_events: open /sys/kernel/debug/tracing/uprobe_events: no such file or directory, (/lib/aarch64-linux-gnu/libc.so.6, 860176)<br>
WAR -- EBPF-DNS: Failed to attach uprobe uretprobe/getaddrinfo : cannot open uprobe_events: open /sys/kernel/debug/tracing/uprobe_events: no such file or directory, (/lib/aarch64-linux-gnu/libc.so.6, 860176)<br>
WAR -- EBPF-DNS: Failed to find symbols for uprobes.<br>
WAR -- EBPF-DNS: Unable to attach ebpf listener: Failed to find symbols for uprobes
</details>

is that normal ?

Thanks.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
For now yes @BobSquarePants . The ebpf DNS module is loaded in a different stage than the ProcMonitorMethod.
